openage/convert/main.py

```diff
@@ -207,7 +214,22 @@ def _get_source_dir_proposals():
         yield "$WINEPREFIX/" + _STANDARD_PATH_IN_64BIT_WINEPREFIX
     yield "~/.wine/" + _STANDARD_PATH_IN_32BIT_WINEPREFIX
     yield "~/.wine/" + _STANDARD_PATH_IN_64BIT_WINEPREFIX
-    # TODO: wine reg query "HKLM\Software\Microsoft\Microsoft Games\Age of Empires\2.0"
+    try:
+        # get wine registry key of the age installation
+        tmp_reg_file = 'aoe_temp.reg'
+        if subprocess.call(['wine', 'regedit', '/E', tmp_reg_file,
+                        _REGISTRY_KEY]) and os.path.exists(tmp_reg_file):
```

Comment:
See here, I did the same thing:
https://github.com/SFTtech/kevin/blob/f978fbd9ee3f1bae8c68b9b1f5266642ff21367c/kevin/util.py#L110
This case is the other way round, python writes it and another process (`ssh`) reads it. Note the context managers of this class.

The complete code would be:
``` python
with NamedTempFile(mode="r") as regfile:
    ret = subprocess.call(['wine', 'regedit', '/E', regfile.name, _REGISTRY_KEY])
    if ret != 0:
        raise whatever
```